test/performance/direct/200-direct-perf.yaml

```diff
@@ -28,6 +28,15 @@ spec:
       labels:
         role: direct-perf-consumer
     spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: kubernetes.io/hostname
+                    operator: In
+                    values:
+                      - node3
```

Comment:
I'm not familiar with this config. Will there be an error if there's no `node3` on the cluster? Does this config support more generic settings?

Reply:
Uops this shouldn't be here, I wrongly committed it, sorry :smile: